security/providers/oidc-common/src/test/java/io/helidon/security/providers/oidc/common/OidcConfigAbstractTest.java

```diff
@@ -73,7 +73,7 @@ void testDefaultValues() {
                   () -> assertThat("Issuer", config.issuer(), nullValue()),
                   () -> assertThat("Client without authentication", config.generalClient(), notNullValue()),
                   () -> assertThat("Client with authentication", config.appClient(), notNullValue()),
-                  () -> assertThat("JWK Keys", config.signJwk(), notNullValue())
+                  () -> assertThat("JWK Keys", config.signJwk(), nullValue())
```

Comment:
We do not return `null` from APIs. The original assertion was correct

Reply:
If there is a chance this is really null, we should throw an exception instead (such as `IllegalStateException("OIDC Configuration is not yet loaded")` or whatever is relevant in this case

Reply:
I have updated JWK value assignment to be more aligned with the previous approach. Now it never returns null.